.isort.cfg

```diff
@@ -4,4 +4,4 @@ include_trailing_comma=True
 force_grid_wrap=0
 use_parentheses=True
 line_length=88
-known_third_party = appdirs,certifi,xattr
+known_third_party = appdirs,certifi,grp,pwd,xattr
```

Comment:
How did these end up in here?

Reply:
seed-isort is a capricious god?

Reply:
TIL- `seed-isort-config` is no longer necessary for `isort>=5.0`, which we're now using.